.github/workflows/ecs.yml

```diff
@@ -109,12 +109,33 @@ jobs:
         docker push $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG--web
         if [ -f $DOCKER_ASSETS_PATH/Dockerfile.${ECR_REPOSITORY}.dj ];
         then
+          HAS_DJ=true
           echo pushing dj
           docker push $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG--dj
         else
           echo not pushing dj
         fi
 
+        echo Pushing 'latest' tags
+        BRANCH=${GITHUB_REF#refs/heads/}
+        if [[ "$BRANCH" =~ ^(production|staging|sandbox|demo|main|release-*|deploy-.*)$ ]]; then
```

Comment:
I think this approach made sense for HNMI, but over here, I think modifying the config/deploy/docker/lib scripts (maybe just deployer.rb) would be better. After that code waits for the build, and before the RollOut class is called, we know the environment and client name, so that should be enough to tag. Let me know, @andyzito if you want to discuss. This will apply to the warehouse as well.

Reply:
@yakloinsteak So to confirm, Open Path (both CAS and Warehouse) don't use the GHA workflow to push images?